env0/resource_template.go

```diff
@@ -440,12 +440,15 @@ func templateRead(prefix string, template client.Template, d *schema.ResourceDat
 	// If this value isn't set, ignore whatever is returned from the response.
 	// This helps avoid drifts when defaulting to 'opentofu' for new 'terragrunt' templates, and 'terraform' for existing 'terragrunt' templates.
 	// 'template.TerragruntTfBinary' field is set to 'omitempty'. Therefore, the state isn't modified if `template.TerragruntTfBinary` is an empty string.
-	if terragruntTfBinary == "" {
-		template.TerragruntTfBinary = ""
-	}
-	// Same explanation as above.
-	if terraformVersion == "" {
-		template.TerraformVersion = ""
+	// This is not true for imports - because the shcema is empty irrespective in that case.
+	if !isImport {
```

Comment:
Is this condition necessary? If the schema is empty anyway
What would be the effect of removing this condition?

Reply:
it will not import terraform_version properly.
If the schema is empty (which it is in import), it will remove these values from the struct, afterwards when updates the schema based on the steuct, terraform_version will not be set.

This occurred to me running the test. So I added this condition.